applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -7,13 +7,28 @@
 menuconfig DESKTOP_DVFS
 	bool "DVFS module"
 	depends on SOC_NRF54H20_CPUAPP
+	select NRFS_DVFS_SERVICE_ENABLED
+	select CLOCK_CONTROL
```

Comment:
We could consider adding a Kconfig dependency to ensure that the used DT alias is defined.

Reply:
Maybe adding:

#if !DT_NODE_HAS_STATUS(DT_ALIAS(nrfdesktop_dvfs_clock), okay)
    #error "Alias 'nrfdesktop_dvfs_clock' is not defined in the device tree!"
#endif

in .c file would be more clear? For me understanding such kconfig error might be confusing

Reply:
`depends on` is better, because it's visible earlier. Build assert should be ok too

Reply:
Disused offline. 

This approach is better: https://github.com/nrfconnect/sdk-nrf/pull/17562#discussion_r1812622730